src/M6Web/Bundle/LogBridgeBundle/Config/FilterParser.php

```diff
@@ -74,19 +81,46 @@ public function parse(string $name, array $config): Filter
             ->setMethod($config['method'])
             ->setStatus($config['status']);
 
-        $this->parseRoute($filter, $config['route']);
+        $this->parseRoutes($filter, $config['routes'] ?? []);
         $this->parseLevel($filter, $config['level']);
 
         return $filter->setOptions($config['options'] ?? []);
     }
 
-    protected function parseRoute(Filter $filter, ?string $route): void
+    protected function parseRoutes(Filter $filter, ?array $routes): void
     {
-        if ($route !== null && !$this->isRoute($route)) {
-            throw new ParseException(sprintf('Undefined route "%s" from router service', $route));
+        if (empty($routes)) {
+            $filter->setRoutes(['all']);
+
+            return;
+        }
+
+        // Find and keep excluded routes
+        $excludedRoutes = array_filter($routes, function (string $route) {
+            return str_contains($route, '!');
```

Comment:
better to check if the first char is '!' ? no ?

Reply:
It's more code, but I understand why, I should trim it also

Reply:
Well using str_contains help me to not have to handle errors if the string left is 0 or something like that. I kinna like it that way 😅